src/win32.c

```diff
@@ -284,7 +284,7 @@ INT CALLBACK BrowseCallbackProc(HWND hwnd, UINT uMsg, LPARAM lp, LPARAM pData)
 gchar *win32_show_folder_dialog(GtkWidget *parent, const gchar *title, const gchar *initial_dir)
 {
 	BROWSEINFOW bi;
-	LPCITEMIDLIST pidl;
+	LPITEMIDLIST pidl;
```

Comment:
L=long
P=pointer
C=const

Just changed it from a const pointer to a non-const one so it could be passed to `CoTaskMemFree()` without warnings/errors.